atlp-rwanda/commanders-rn-medica
# feature: Implementing Local Security Features

#### What does this PR do?
This PR implements local security features in the Medica app to enhance user account protection and provide convenient access through biometric authentication and other security options.

#### Description of Task to be completed?
- Implement "Remember me" functionality for persistent login sessions.
- Integrate biometric authentication (Face ID and/or Biometric ID) using Expo Local Authentication.
- Implement PIN change functionality for enhanced account security.

#### How should this be manually tested?
1. Navigate to the Security section in the app settings.
2. Toggle the "Remember me" option and verify if login sessions persist across app restarts.
3. Enable Face ID and Biometric ID authentication options and verify if they work correctly on supported devices.
5. Enable PIN authentication options and verify if they work correctly on supported devices.
